core/db/migrate/20220103082046_add_status_and_make_active_at_to_spree_products.rb

```diff
@@ -1,7 +1,9 @@
 class AddStatusAndMakeActiveAtToSpreeProducts < ActiveRecord::Migration[5.2]
   def change
     add_column :spree_products, :status, :string, null: false, default: 'draft'
+    add_column :spree_products, :make_active_at, :datetime
```

Comment:
@kgorazd let's not modify existing migrations, let's add a new one please

Reply:
done